src/llama-graph.cpp

```diff
@@ -907,31 +907,38 @@ ggml_tensor * llm_graph_context::build_moe_ffn(
         cb(cur, "ffn_moe_weighted", il);
     }
 
-    ggml_tensor * up = build_lora_mm_id(up_exps, cur, selected_experts); // [n_ff, n_expert_used, n_tokens]
-    cb(up, "ffn_moe_up", il);
+    ggml_tensor * tmp = build_lora_mm_id(up_exps, cur, selected_experts); // [n_ff, n_expert_used, n_tokens]
```

Comment:
I think we can still call this `up`, right? There is no other places where we re-assign another value for `tmp`

Reply:
The only reason to call it `tmp` would be that that's what (non-moe) build_ffn calls it, which makes it easier to compare the two functions. But `up` is fine.